src/framework/mlt_slices.h

```diff
@@ -38,4 +39,12 @@ extern int mlt_slices_count( mlt_slices ctx );
 
 extern mlt_slices mlt_slices_init_pool( int threads, int policy, int priority, const char* name );
 
+extern mlt_slices mlt_slices_get_global( mlt_schedule_policy policy );
```

Comment:
mlt_slices_count is used by frei0r, which requires to know the number of slices ahead of mlt_slices callback function to set dimension of slice on frei0r API.